sections/email-signup-banner.liquid

```diff
@@ -157,23 +172,67 @@
     },
     {
       "type": "select",
-      "id": "desktop_text_box_position",
+      "id": "text_alignment",
       "options": [
         {
-          "value": "flex-start",
-          "label": "t:sections.email-signup-banner.settings.desktop_text_box_position.options__1.label"
+          "value": "left",
+          "label": "t:sections.image-banner.settings.text_alignment.options__1.label"
         },
         {
           "value": "center",
-          "label": "t:sections.email-signup-banner.settings.desktop_text_box_position.options__2.label"
+          "label": "t:sections.image-banner.settings.text_alignment.options__2.label"
         },
         {
-          "value": "flex-end",
-          "label": "t:sections.email-signup-banner.settings.desktop_text_box_position.options__3.label"
+          "value": "right",
+          "label": "t:sections.image-banner.settings.text_alignment.options__3.label"
         }
       ],
       "default": "center",
-      "label": "t:sections.email-signup-banner.settings.desktop_text_box_position.label"
+      "label": "t:sections.image-banner.settings.text_alignment.label"
+    },
+    {
```

Comment:
Not mobile text alignment option here ?

Reply:
Yes, let me update that. This was a new update and I forgot to add it here 👍